$OpenBSD: patch-Makefile,v 1.2 2017/11/03 12:44:16 espie Exp $
- obey CC/CFLAGS
- explicitly pass -lm for gromit as it uses directly

Index: Makefile
--- Makefile.orig
+++ Makefile
@@ -1,10 +1,10 @@
 all: gromit
 
 proptest: proptest.c
-	gcc -o proptest proptest.c `gtk-config --libs --cflags`
+	${CC} ${CFLAGS} -o proptest proptest.c `gtk-config --libs --cflags`
 
 propertywatch: propertywatch.c
-	gcc -o propertywatch propertywatch.c `gtk-config --libs --cflags`
+	${CC} ${CFLAGS} -o propertywatch propertywatch.c `gtk-config --libs --cflags`
 
 gromit: gromit.c Makefile
-	gcc -DG_DISABLE_DEPRECATED -DGDK_PIXBUF_DISABLE_DEPRECATED -DGDK_DISABLE_DEPRECATED -DPANGO_DISABLE_DEPRECATED -DGDK_MULTIHEAD_SAFE -DGTK_MULTIHEAD_SAFE -o gromit gromit.c -Wall `pkg-config --libs --cflags gtk+-2.0`
+	${CC} ${CFLAGS} -DG_DISABLE_DEPRECATED -DGDK_PIXBUF_DISABLE_DEPRECATED -DGDK_DISABLE_DEPRECATED -DPANGO_DISABLE_DEPRECATED -DGDK_MULTIHEAD_SAFE -DGTK_MULTIHEAD_SAFE -o gromit gromit.c -Wall `pkg-config --libs --cflags gtk+-2.0` -lm
